README.md

```diff
@@ -0,0 +1,96 @@
+<br>
+<br>
+
+<p align="center">
+    <img src="docs/logo-full.svg" alt="oppskrift-logo">
+</p>
+
+<p align="center">
+    <strong>A decentralized network for cooking lovers</strong>
+</p>
+
+<br>
+<br>
+
+<p align="center">
+    <!-- TODO : choose a licence <img alt="license-badge" src="https://img.shields.io/github/license/Oppskrift/oppskrift_api_sf?style=for-the-badge"> -->
```

Comment:
You can go for GPL 3 I think, I did so for the web app

Reply:
It's done :)